internal/hns/hnsendpoint.go

```diff
@@ -31,6 +31,7 @@ type HNSEndpoint struct {
 	EnableLowMetric    bool              `json:",omitempty"`
 	Namespace          *Namespace        `json:",omitempty"`
 	EncapOverhead      uint16            `json:",omitempty"`
+	SharedContainers   []string          `json:",omitempty"`
```

Comment:
What is this change for?

Reply:
In kubelet, I need to know which containers the endpoint is associated with.  I found this function: https://github.com/microsoft/hcsshim/blob/9bc76cd068a2c05a76f678253b5d4dad27629f9c/internal/hns/hnsendpoint.go#L106 that does this check so I am exposing the data here.